deployments/properties/azure-kubernetes-service-helm.properties.json

```diff
@@ -0,0 +1,7 @@
+{
+    "name": "Deploy to AKS with Helm",
+    "description": "Deploy an application to an Azure Kubernetes Service Cluster using Azure Credentials and Helm",
+    "creator": "Microsoft Azure",
+    "iconName": "azure",
+    "categories": ["Deployment", "Helm", "Kubernetes", "Dockerfile"]
```

Comment:
@anuragc617 should we have convention of having the language specified at 2nd position just after type of template?